### PR TITLE
Pub sub fix

### PR DESCRIPTION
**Pull request by Steven0351**
Addresses issue with subscribers receiving all messages for all topics

### CODE REVIEW

**Review by giralte-ionic — CHANGES_REQUESTED** [review]
please fix the Github Lint warnings as I assume != and !== mean two different things

**Comment by Steven0351** [comment]
> please fix the Github Lint warnings as I assume != and !== mean two different things

Well, looks like at some point i'm going to make the linter error on warnings. They are different though I'm not sure it really matters when checking against undefined. The difference is != will coerce and !== will not coerce.

File: example/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "format": "prettier App.tsx -w"
   },
   "dependencies": {
-    "@ionic/portals-react-native": "^0.0.1",
+    "@ionic/portals-react-native": "file:../ionic-portals-react-native-0.0.1.tgz",
```
Comment (carlpoole) [inline example/package.json:14]: Was this intended to replace with a local zip

**Review by giralte-ionic — APPROVED** [review]

File: example/package.json (hunk 1 above)
Comment (Steven0351) [reply example/package.json:14]: My intention is to reconfigure the project to have a monorepo structure where the example project is not a subfolder of the library itself

**Pull request merged**